src/pybind/mgr/volumes/fs/subvolume.py

```diff
@@ -133,80 +55,80 @@ def _mkdir_p(self, path, mode=0o755):
             except cephfs.ObjectNotFound:
                 self.fs.mkdir(subpath, mode)
 
-    def create_group(self, group_id, mode=0o755):
-        path = self._group_path(group_id)
-        self._mkdir_p(path, mode)
-
-    def delete_group(self, group_id):
-        path = self._group_path(group_id)
-        self.fs.rmdir(path)
+    def _list_dir(self, dir_path):
```

Comment:
I don't think we need this method in this PR

Reply:
yeh -- wil remove.